nucypher/config/keyring.py

```diff
@@ -510,8 +515,8 @@ def derive_crypto_power(self, power_class: ClassVar) -> Union[KeyPairBasedPower,
     def generate(cls,
                  checksum_address: str,
                  password: str,
-                 encrypting: bool,
-                 rest: bool,
+                 encrypting: bool = True,
+                 rest: bool = False,
                  host: str = None,
                  curve: EllipticCurve = None,
                  keyring_root: str = None,
```

Comment:
This is a bit awkward. Seems a bit odd to take an assumption on _what_ the keypair will do (e.g. encrypting), but not the curve to do it with.